core/tests.py

```diff
@@ -0,0 +1,15 @@
+# coding: utf-8
+from django.test import TestCase
+
+
+class AboutpageTest(TestCase):
+    def setUp(self):
+        self.resp = self.client.get('/about/')
+
+    def test_get(self):
+        'GET / must return status code 200'
```

Comment:
I understand your point but I do not understand what are you proposing. Can you show me an example, please?